timescaledb/how-to-guides/compression/modify-a-schema.md

```diff
@@ -1,19 +1,45 @@
 # Schema modifications
-In TimescaleDB 2.1 and later, you can modify the table definition for hypertables with compressed chunks by adding nullable columns, and renaming existing columns.
+You can modify the schema of compressed hypertables in recent versions of
+TimescaleDB.
 
-## Add nullable columns
+|Schema modification|Before TimescaleDB&nbsp;2.1|TimescaleDB&nbsp;2.1 to 2.5|TimescaleDB&nbsp;2.6 and above|
+|-|-|-|-|
+|Add a nullable column|❌|✅|✅|
+|Add a column with a default value and a `NOT NULL` constraint|❌|❌|✅|
+|Rename a column|❌|✅|✅|
+|Drop a column|❌|❌|✅|
+|Change the data type of a column|❌|❌|❌|
+
+To perform operations that aren't supported on compressed hypertables, first
+[decompress][decompression] the table.
+
+## Add a nullable column
 To add a nullable column:
 ```sql
 ALTER TABLE <hypertable> ADD COLUMN <column_name> <datatype>;
 ```
-Note that adding constraints to the new column is not supported.
 
 For example:
 ```sql
 ALTER TABLE conditions ADD COLUMN device_id integer;
 ```
+Note that adding constraints to the new column is not supported before
+TimescaleDB 2.6.
+
+## Add a column with a default value and a NOT NULL constraint
```

Comment:
I wish this worked! 😾 Markdown formatting doesn't work in headings right now, I'm going to put it on the list to look into because I really want to have inline code in headings

Reply:
Oh, bummer. I thought it did 😢